modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/synapserelease/v0.1.1.json

```diff
@@ -0,0 +1,60 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/synapserelease/v0.1.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SynapseReleaseShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "targetClass": "nsg:SynapseRelease",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:distribution",
+              "name": "Synapse release data location",
+              "description": "Synapse release data location.",
+              "node": "this:SynapseReleaseDistributionShape"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "this:SynapseReleaseDistributionShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:mediaType",
+              "name": "Media type",
+              "description": "",
+              "in": [
+                "application/mod"
```

Comment:
application/vdn.bbp.mod ?

Reply:
this is not a bbp thing here.
it is a Neuron simulator type of file

Reply:
OKay.